webapp/src/main/java/com/box/l10n/mojito/service/tm/TMTextUnitRepository.java

```diff
@@ -53,4 +54,7 @@ public interface TMTextUnitRepository extends JpaRepository<TMTextUnit, Long> {
   @Query(
       "SELECT t FROM TMTextUnit t JOIN FETCH t.asset a JOIN FETCH a.repository JOIN FETCH t.tm WHERE t.id = :id")
   Optional<TMTextUnit> findByIdWithAssetAndRepositoryAndTMFetched(@Param("id") Long id);
+
+  @Procedure(procedureName = "fetch_text_unit_count_for_appending")
```

Comment:
We should add the procedure creation to the migration scripts under webapp/src/main/resources/db/migration if we're referencing it in the code